autosklearn/evaluation/__init__.py

```diff
@@ -85,7 +91,8 @@ def get_cost_of_crash(metric):
     return worst_possible_result
 
 
-def _encode_exit_status(exit_status):
+def _encode_exit_status(exit_status: multiprocessing.connection.Connection
```

Comment:
I'm honestly a bit surprised about this type annotation? Maybe we should annotate the pynisher first?

Reply:
We saw this in autopytorch. I think it makes sense to annotate. Here because we are limited to just evaluation, I can define it as int/str directly.